packages/perseus/src/widgets/interactive-graphs/mafs-graph.test.tsx

```diff
@@ -63,3 +68,192 @@ describe("MafsGraph", () => {
         expect(mockChangeHandler).toHaveBeenCalled();
     });
 });
+
+function map(
```

Comment:
This is really common functionality. In [P5](https://p5js.org/reference/#/p5/map) it's called `map`; I think Math speak it's called a linear transform. I left this here basically to ask if anyone knows if we have this already (maybe in underscore/lodash/jquery with a different name?)

Reply:
Example: `map(2, 0, 10, 0, 100)` scales what would be 2 in a 0-10 range into 20 in a 0-100 range.

Reply:
I wonder if we should call the function something like `linearTransform` (or something similar) so that we don't imply a meaning that isn't there due to the JavaScript `Array.prototype.map` function.

Reply:
Mafs exports utilities for creating and applying transformation matrices, which I think could also be used to implement `graphToPixel`. E.g.

```ts
const graphToPixelTransform = vec.matrixBuilder().translate(-rangeMin, 0)
  .scale(fullPixelSpace / (rangeMax - rangeMin)).translate(halfPixelSpace);

vec.transform(point, graphToPixelTransform);
```

I don't think that's the right logic, but something like that structure could work.

Reply:
@benchristel Done